src/main/scala/ai/nikin/typedgraph/core/Path.scala

```diff
@@ -0,0 +1,12 @@
+package ai.nikin.typedgraph.core
+
+class Path(edges: Set[Edge]) {
```

Comment:
Are we sure we want this to be `Set`? The default implementation for `Set()` is `HashSet()` so it can break the order.

Reply:
Yes Fran said the same thing, it is fixed now